doc/INSTALL.md

```diff
@@ -255,11 +255,18 @@ If you need SQLite (or get a SQLite mismatch build error):
     $ brew install sqlite
     $ export LDFLAGS="-L/usr/local/opt/sqlite/lib"
     $ export CPPFLAGS="-I/usr/local/opt/sqlite/include"
+    
+For M1 macs: 
+
+    $ brew install sqlite
+    $ export LDFLAGS="-L/opt/homebrew/opt/sqlite/lib"
+    $ export CPPFLAGS="-I/opt/homebrew/opt/sqlite/include"
```

Comment:
I wonder if rather than documenting this here we should fix the `Makefile` where we have a partial detection in place already:

https://github.com/ElementsProject/lightning/blob/21cc16fb5bfe116bd0430a19c1301ef606c424b9/Makefile#L236-L242